cmd/protecodeExecuteScan_test.go

```diff
@@ -399,13 +399,23 @@ func TestGetTarName(t *testing.T) {
 			"3.20.20-20200131085038+eeb7c1033339bfd404d21ec5e7dc05c80e9e985e",
 			"abc_3.tar",
 		},
-		"without version ": {
+		"without version": {
 			"abc",
 			"",
 			"abc.tar",
 		},
+		"ScanImage without sha as artifactVersion": {
+			"abc@sha256:12345",
+			"",
+			"abc.tar",
+		},
+		"ScanImage with sha as artifactVersion": {
+			"ppiper/cf-cli@sha256:c25dbacb9ab6e912afe0fe926d8f9d949c60adfe55d16778bde5941e6c37be11",
+			"c25dbacb9ab6e912afe0fe926d8f9d949c60adfe55d16778bde5941e6c37be11",
+			"ppiper_cf-cli_c25dbacb9ab6e912afe0fe926d8f9d949c60adfe55d16778bde5941e6c37be11.tar",
+		},
 	}
-
```

Comment:
I would like to suggest not to introduce the tab/spaces here

Reply:
@marcusholl I reintroduced the space, because initially there was space.
If you insist - I can take it remove it.